pdsgrouptable.py

```diff
@@ -170,6 +171,10 @@ def insert(self, things):
 
         elif isinstance(thing, pdsfile.PdsFile):
             self.insert_file(thing)
+        elif (isinstance(thing, pdsfile.PdsFile) or
```

Comment:
The first `isinstance` can never be hit, can it? Because it would be hit in the clause above. I think you want to delete the clause above.
For the others, maybe use `issubclass` instead.

Reply:
Fixed